cmd/system-probe/api/module/loader.go

```diff
@@ -174,11 +175,18 @@ func updateStats() {
 	then := time.Now()
 	now := time.Now()
 	ticker := time.NewTicker(15 * time.Second)
+	health := health.RegisterLiveness("system-probe")
+
+	log.Info("system probe: health check initialized successfully")
```

Comment:
The health check cannot error on register, so I'm not sure this log is valuable.